documentation/source/user_section/tutorials/multimodal-registration/contrast-agnostic-registration/coregistering-t1-t2.rst

```diff
@@ -1,3 +1,8 @@
+..
+   comment:: This section is no longer included in the SCT course to save on
+             time. However, we still keep it in the tutorials because it is
+             relatively unobtrusive and may be useful to some users.
```

Comment:
re: d2281096634966b7115733ebfe72d3250192a801; We decided we wanted to keep `algo=dl` in the tutorials, right? (todo: dig up meeting minutes)